control/veda_deliver.py

```diff
@@ -5,10 +5,10 @@
 
 """
 import datetime
+import ftplib
 import logging
 import shutil
 from os.path import expanduser
-import sys
```

Comment:
We are using `sys` in the code below.

Reply:
its coming from `from control_env import *`